### PR TITLE
Resolving bad merge with cherry-pick (#964)

### PR DESCRIPTION
**Pull request by RichieEscarez**
Update traffic splitting example to use Release mode (#964) - Also applies to v0.3, v0.4

* Update traffic splitting example to use Release mode

This change updates the traffic splitting example to use release mode
introducing the concepts of the mode in three steps. This change also
simplifies running the commands and removes the need to mutate files on
disk to run through the example.

* Use envsubst instead of perl. Simplify image path

* Use long names for arguments

* Update diff statement

* Add envsubst install instructions

### CODE REVIEW

File: docs/serving/samples/traffic-splitting/updated_configuration.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 The Knative Authors
```
Comment (dgerd) [inline docs/serving/samples/traffic-splitting/updated_configuration.yaml]: This file should be deleted. It is the same as `release_sample.yaml` above

**Comment by RichieEscarez** [comment]
duplicate file removed, ptal

**Pull request merged**